article/app/views/liveblog/liveBlogBodyContent.scala.html

```diff
@@ -21,7 +22,23 @@
             _root_.liveblog.LatestBlock(article.blocks)
         }" data-test-id="live-blog-blocks"
         itemprop="@if(article.tags.isReview) {reviewBody} else {articleBody}">
-        @views.html.liveblog.liveBlogBlocks(model.currentPage.currentPage.blocks, article, timezone, amp)
+            @if(amp) {
+                @model.currentPage.currentPage.blocks.sliding(5, 5).zipWithIndex.map { case (blockGroup, index) =>
+                    @views.html.liveblog.liveBlogBlocks(blockGroup, article, timezone, amp)
+
+                    @* Add advert every 5 blog posts, up to a maximum of 8 *@
+                    @if(blockGroup.length == 5 && index < 8) {
```

Comment:
Perhaps we don't need a maximum then? I was using the rules defined in [liveblog-adverts.js](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/liveblog-adverts.js#L24)